__mocks__/@react-native-community/async-storage.js

```diff
@@ -0,0 +1,37 @@
+let cache = {
+  '@User:tipsShown': 'true'
+};
+
+/* eslint-disable */
+export const changeCache = object => {
+  cache = object;
+};
+
+export default {
+  setItem: (key, value) => {
+    return new Promise((resolve, reject) => {
+      return typeof key !== 'string' || typeof value !== 'string'
+        ? reject(new Error('key and value must be string'))
+        : resolve((cache[key] = value));
+    });
+  },
+  getItem: (key, value) => {
+    return new Promise(resolve => {
+      return cache.hasOwnProperty(key) ? resolve(cache[key]) : resolve(null);
+    });
+  },
+  removeItem: key => {
+    return new Promise((resolve, reject) => {
+      return cache.hasOwnProperty(key) ? resolve(delete cache[key]) : reject('No such key!');
+    });
+  },
+  clear: key => {
+    return new Promise((resolve, reject) => resolve((cache = {})));
+  },
+
```

Comment:
This `async-storage` mock is not always necessary. Only if you want to handle a lot of logic with it.